content/docs/programs/requisitions.md

```diff
@@ -73,6 +73,55 @@ The list view has gained some additional columns, showing the program related da
 - **Order type**: the name of the order type ( typically, a normal or emergency order )
 - **Period**: the name of the period selected
 
-And the detail view also has program information shown:
+#### Manual Program Requisitions
+
+You can create a manual program requisition for customers who do not use mSupply
+and those who do (for those who do, this should only be done in exceptional).
```

Comment:
```suggestion
Program Requisitions can also be created manually. This would typically be for customers who do not use mSupply, or in exceptional circumstances.
```